fbchat/_client.py

```diff
@@ -976,6 +976,41 @@ def getUserActiveStatus(self, user_id):
         """
         return self._buddylist.get(str(user_id))
 
+    def fetchThreadImages(self, thread_id=None):
+        """
+        Creates generator object for fetching images posted in thread.
+        :param thread_id: ID of the thread
+        :return: :class:`ImageAttachment` or :class:`VideoAttachment`.
+        :rtype: iterable
+        """
+        thread_id, thread_type = self._getThread(thread_id, None)
+        data = {"id": thread_id, "first": 48}
+        j = self.graphql_request(_graphql.from_query_id("515216185516880", data))
+        while True:
+            try:
+                i = j[thread_id]["message_shared_media"]["edges"][0]
```

Comment:
The ID is a string key in the response, so please make sure to convert the `thread_id` before using it here (so that the user can pass an `int`, and it'll still work fine)

Reply:
> so that the user can pass an int, and it'll still work fine

Oh, yeah, good point.